Traceback (most recent call last):
  File "miagi.py", line 85, in <module>
    bot.new_auto_mod()
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 726, in new_auto_mod
    time.sleep(3)
KeyboardInterrupt
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 343, in cleanup
    time.sleep(sleeptime)
KeyboardInterrupt
Traceback (most recent call last):
  File "miagi.py", line 85, in <module>
    bot.new_auto_mod()
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 726, in new_auto_mod
    time.sleep(3)
KeyboardInterrupt
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 343, in cleanup
    time.sleep(sleeptime)
KeyboardInterrupt
Traceback (most recent call last):
  File "miagi.py", line 85, in <module>
    bot.new_auto_mod()
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 726, in new_auto_mod
    time.sleep(3)
KeyboardInterrupt
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 343, in cleanup
    time.sleep(sleeptime)
KeyboardInterrupt
Traceback (most recent call last):
  File "miagi.py", line 85, in <module>
    bot.new_auto_mod()
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 726, in new_auto_mod
    time.sleep(3)
KeyboardInterrupt
Traceback (most recent call last):
  File "miagi.py", line 85, in <module>
    bot.new_auto_mod()
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 726, in new_auto_mod
    time.sleep(3)
KeyboardInterrupt
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 343, in cleanup
    time.sleep(sleeptime)
KeyboardInterrupt
Traceback (most recent call last):
  File "miagi.py", line 85, in <module>
    bot.new_auto_mod()
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 726, in new_auto_mod
    time.sleep(3)
KeyboardInterrupt
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 343, in cleanup
    time.sleep(sleeptime)
KeyboardInterrupt
Traceback (most recent call last):
  File "miagi.py", line 85, in <module>
    bot.new_auto_mod()
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 726, in new_auto_mod
    time.sleep(3)
KeyboardInterrupt
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/home/ubuntu/Downloads/pyfund/instabot.py/src/instabot.py", line 343, in cleanup
    time.sleep(sleeptime)
KeyboardInterrupt
